templates/kratos.yaml.j2

```diff
@@ -27,13 +27,13 @@ selfservice:
         login:
             ui_url: {{ login_ui_url }}
         {%- endif %}
+        {%- if oidc_providers %}
         registration:
-            {%- if oidc_providers %}
             after:
                 oidc:
                     hooks:
                     - hook: session
-            {%- endif %}
+        {%- endif %}
     {%- endif %}
```

Comment:
This block depends on `{%- if error_ui_url or login_ui_url or registration_ui_url %}`, but there is no `registration_ui_url` property in the kratos charm. Should we change it to `oidc_providers`?
```yaml
{%- if error_ui_url or login_ui_url or oidc_providers %}
flows:
    {%- if error_ui_url %}
    error:
        ui_url: {{ error_ui_url }}
    {%- endif %}
    {%- if login_ui_url %}
    login:
        ui_url: {{ login_ui_url }}
    {%- endif %}
    {%- if oidc_providers %}
    registration:
        after:
            oidc:
                hooks:
                - hook: session
    {%- endif %}
{%- endif %}
```

Reply:
Yes, good catch.